### DISCUSSION TITLE
Problems with subscribe() in @supabase/supabase-js 1.35.3

### DISCUSSION THREAD

**Discussion by tomsoderlund** [supabase/supabase#12582] [Questions]
Hi, can you help me understand why this `subscribe` works great:

    // Subscribe to updates to the current article
    useEffect((): any => {
      if (articleId === null) return
      const articleChangeSubscription = supabase
        .from(`article:id=eq.${articleId ?? ''}`)
        .on('UPDATE', updateArticle)
        .on('INSERT', updateArticle)
        .on('DELETE', updateArticle)
        .subscribe()
      const cancelSubscription = async (): Promise<SupabaseSubscriptionResults> => await supabase.removeSubscription(articleChangeSubscription)
      return cancelSubscription
    }, [articleId])

but this doesn't work at all (`updateUserArticlesNoResults` is never triggered):

    // Subscribe to updates to the current user’s articles
    useEffect((): any => { // TODO: better return type
      if (session?.user?.id === null) return
      const myArticlesChangeSubscription = supabase
        .from(`article:user_id=eq.${session?.user?.id ?? ''}`)
        .on('UPDATE', updateUserArticlesNoResults)
        .on('INSERT', updateUserArticlesNoResults)
        .on('DELETE', updateUserArticlesNoResults)
        .subscribe()
      const cancelSubscription = async (): Promise<SupabaseSubscriptionResults> => await supabase.removeSubscription(myArticlesChangeSubscription)
      return cancelSubscription
    }, [session?.user?.id])

Thanks!

**Comment by mansueli** [comment]
Can you try mocking your second subscription with this Realtime Inspector?  

https://realtime.supabase.com/inspector/new

(you might need to dump session?.user?.id to the console to simulate it)
